src/raftstore/store/snap.rs

```diff
@@ -254,9 +255,12 @@ mod v1 {
     /// `save` method to make them persistent. When saving a crc32 checksum
     /// will be appended to the file end automatically.
     pub struct Snap {
+        mgr: SnapManager,
```

Comment:
It needs only a global map with lock here. This part of code could be refactored to use `Arc<RwLock< Map` like `size_track`.